src/clients/Governance.ts

```diff
@@ -458,26 +459,64 @@ export class Governance extends API {
     return result.data
   }
 
-  async getValidationMessage() {
+  async getValidationMessage(account?: AccountType) {
+    const params = new URLSearchParams()
+    if (account) {
+      params.append('account', account)
+    }
     const result = await this.fetch<ApiResponse<string>>(
-      '/user/validate',
+      `/user/validate?${params.toString()}`,
       this.options().method('GET').authorization({ sign: true })
     )
 
     return result.data
   }
 
-  async validateProfile() {
+  async validateForumProfile() {
+    const result = await this.fetch<ApiResponse<{ valid: boolean }>>(
+      '/user/validate/forum',
+      this.options().method('POST').authorization({ sign: true })
+    )
+
+    return result.data
+  }
+
+  async validateDiscordProfile() {
     const result = await this.fetch<ApiResponse<{ valid: boolean }>>(
-      '/user/validate',
+      '/user/validate/discord',
       this.options().method('POST').authorization({ sign: true })
     )
 
     return result.data
   }
 
-  async isProfileValidated(address: string) {
-    const result = await this.fetch<ApiResponse<boolean>>(`/user/${address}/is-validated`, this.options().method('GET'))
+  async isProfileValidated(address: string, accounts: AccountType[]) {
+    const params = new URLSearchParams()
+    for (const account of accounts) {
+      params.append('account', account)
+    }
+    const result = await this.fetch<ApiResponse<boolean>>(
+      `/user/${address}/is-validated/?${params.toString()}`,
+      this.options().method('GET')
+    )
+
+    return result.data
+  }
+
+  async getIsDiscordActive() {
```

Comment:
`isDiscordActive`? If we go for `getIsDiscordActive` maybe we should update `isProfileValidated` up here too.